dataminer/DataMiner_Tools/BrokerGateway_Migration.md

```diff
@@ -141,55 +144,54 @@ The following actions will be executed during the migration, in the indicated or
+No. Ports conflict and credentials differ; mixed setups do not function.
 
-### Why should I migrate to BrokerGateway? What are the advantages?
+### Why should I migrate to BrokerGateway?
 
-BrokerGateway will manage NATS communication based on a single source of truth that has the complete knowledge of the cluster, resulting in more robust, carefree NATS communication. In addition, TLS will be configured automatically, and a newer version of NATS will be used that has better performance and is easier to upgrade.
+It provides a single authoritative cluster view, automatic TLS, simplified lifecycle management and improved performance vs the legacy SLNet‑managed setup.
+From DataMiner 10.6.0 onwards the SLNet‑managed NATS solution is no longer available and migration before upgrading is mandatory.
 
 ## Troubleshooting
 
 ### ERROR: {machineName} was not able to remove itself from its current cluster in order to join the new cluster
 
-When calling *ResetCluster*, BrokerGateway will first try to remove itself from any cluster it is part of, in order to set up a new cluster with all the endpoints specified in `C:\Skyline DataMiner\Configurations\ClusterEndpoints.json`.
-
-The endpoints BrokerGateway is currently clustered with are listed in the ClusterInfo in `C:\Program Files\Skyline Communications\DataMiner BrokerGateway\appsettings.runtime.json`. If one of those IPs cannot be reached, the error message above is generated.
-
-You will be asked if the node may be forcibly removed. If you agree to this, the current machine can free itself from the cluster to cluster with the new setup instead. However, because it forcibly removes itself, it no longer informs any of the unreachable endpoints of its removal. These may still attempt to contact the current machine, which will no longer be reachable for them. If you choose to not allow the forcible removal, this will cancel the migration process and revert back to the previous configuration.<!-- RN 40991 -->
+BrokerGateway attempts to detach from any existing cluster (endpoints in `ClusterEndpoints.json`). If an endpoint is unreachable a prompt for forced removal appears. Forced removal proceeds without informing unreachable peers. Declining cancels and reverts.<!-- RN 40991 -->
 
-If you do want to migrate to BrokerGateway but you do not want this forced removal, make sure all endpoints specified in *appsettings.runtime.json* can be reached by the current machine.
+Ensure all endpoints listed in `appsettings.runtime.json` are reachable before retrying.
 
 ### NATSRepair.exe
 
-If you encounter issues with your NATS cluster and you have migrated to BrokerGateway, you can run the *NATSRepair.exe* tool from the `C:\Skyline DataMiner\Tools\` folder. This will perform a repair on the cluster.<!-- RN 42328 -->
+If issues occur after migration run `NATSRepair.exe` from `C:\Skyline DataMiner\Tools\` to repair the cluster.<!-- RN 42328 -->
```

Comment:
ok to run this on just one of the agents, right?
which types of issues should these be used for?